Automattic/jetpack
# Instant Search: Render disabled filter widgets

## Changes proposed in this Pull Request:
When a user performs a search using a Jetpack Search widget, the search page will now include a dynamically generated list of filters on the sidebar like so:

> <img width="346" alt="Screen Shot 2019-08-26 at 4 18 28 PM" src="https://user-images.githubusercontent.com/4044428/63727314-29a81980-c81d-11e9-8584-371c74ca18de.png">

The checkbox inputs are currently disabled at this time. Instant Search remains gated by a `JETPACK_SEARCH_PROTOTYPE` define.

## Is this a new feature or does it add/remove features to an existing part of Jetpack?
* Yes, this is a new feature being added onto Jetpack Instant Search (p3QzjZ-V3).

## Testing instructions:
* Add `define( "JETPACK_SEARCH_PROTOTYPE", true );` to your wp-config.php.
* Ensure that your site has the Jetpack Pro plan and has Jetpack Search enabled.
* Add a Jetpack Search widget to the Search page sidebar.
* Add a filter or two to the Jetpack Search widget.
* Enter a query into a search widget. 
Alternatively, navigate to a search page like `/?s=privacy`.
* Ensure that you see a list of disabled checkboxes in the Search page sidebar. 

## Proposed changelog entry for your changes:
* No changelog entry necessary; these changes will not be accessible to our users.
